chatsky/context_storages/memory.py

```diff
@@ -0,0 +1,93 @@
+import asyncio
+from typing import Dict, List, Optional, Set, Tuple, Hashable
+
+from .database import DBContextStorage, FieldConfig
+
+
+class MemoryContextStorage(DBContextStorage):
+    """
+    Implements :py:class:`.DBContextStorage` storing contexts in memory, wthout file backend.
+    Uses :py:class:`.JsonSerializer` as the default serializer.
+    By default it sets path to an empty string.
+
+    Keeps data in a dictionary and two lists:
+    
+    - `main`: {context_id: [created_at, turn_id, updated_at, framework_data]}
+    - `turns`: [context_id, turn_number, label, request, response]
```

Comment:
Did we decide to store labels, requests and responses in the same table?
Because of this requests are loaded with an extra element (request 0 which was never added, but turn 0 exists).
Tutorial 6_context_serialization fails because of it (it counts the number of requests).